test/unit/gossip-client-daemonset.bats

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env bats
+
+load _helpers
+
+@test "global/gossipEncryption: disabled in client DaemonSet by default" {
```

Comment:
It would be good to check that the `-encrypt` flag isn't present in this case as well.

Reply:
Accepted your change